modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -980,7 +980,11 @@ bool CvCapture_FFMPEG::open(const char* _filename, const VideoCaptureParameters&
     char* options = getenv("OPENCV_FFMPEG_CAPTURE_OPTIONS");
     if(options == NULL)
     {
+#if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(55, 48, 100)
```

Comment:
> 100

It is FFmpeg-specific version component. Original libav has different versions and we don't want to enable this code for libav (e.g. `(55, 49, 0)` version).
Please add `LIBAVUTIL_VERSION_MICRO >= 100` check to ensure that it is FFmpeg fork (see below).

Reply:
> It is FFmpeg-specific version component. Original libav has different versions and we don't want to enable this code for libav (e.g. `(55, 49, 0)` version).

I did not know that, thanks, changed to ensure we are cheking the ffmpeg version of libavutil.